lib/ProductOpener/Ingredients.pm

```diff
@@ -3382,8 +3382,8 @@ sub normalize_enumeration ($lc, $type, $enumeration) {
 		$trailing_space = " ";
 	}
 
-	my $and = $Lang{_and_}{$lc};
-	#my $enumeration_separators = $obrackets . '|' . $cbrackets . '|\/| \/ | ' . $dashes . ' |' . $commas . ' |' . $commas. '|'  . $Lang{_and_}{$lc};
+	# do not match anything if we don't have a translation for "and"
+	my $and = $and{$lc} || " will not match ";
```

Comment:
it's a bit confusing to have `$and` named as `%and`… but it's ok.

Also we don't support a lot of languages in `%and` for now… shan't you report current translations in %and ?

`grep -A 3 'msgctxt "_and_"' -r po/common/ |grep msgstr`

Reply:
@alexgarel Nice, I never used the -A option :)

Reply:
I added some of the translations, and left out the ones for languages we have poor support (non roman characters mostly)